pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1156,7 +1156,22 @@ self: super: {
 
   xmonad-extras = doJailbreak super.xmonad-extras;
 
-  arbtt = doJailbreak super.arbtt;
+  arbtt = overrideCabal super.arbtt (drv: {
```

Comment:
This is way too complicated. Just copy the relevant bit of `haskell-packages.nix` that updates `arbtt` from https://github.com/NixOS/nixpkgs/commit/e98e4d21fa6744da13cd08fd4ebf585fdcdb19c2 into the release branch.

Reply:
So I should make changes in `hackage-packages.nix`? I thought that file is supposed to be generated only. I'll update the PR.

Reply:
If we ever re-generate it, the new file will contain the update that you're merging here ... so in this case the manual edit is fine. It would not be fine in `master`, though.